packages/run-protocol/src/reserve/doc.md

```diff
@@ -0,0 +1,129 @@
+# Reserve 
+
+### addIssuer
+
+```mermaid
+sequenceDiagram
+
+participant caller
+participant addIssuer
+participant ammPublicFacet
+participant brandForKeyword
+participant keywordForBrand
+participant zcf
+
+
+caller ->> addIssuer: addIssuer(inIssuer, inKeyword)
```

Comment:
not sure what `in` means here. `addIssuer` has no constraint on the what the issuer is used for.
```suggestion
caller ->> addIssuer: addIssuer(issuer, keyword)
```